### PR TITLE
Edited fedora code to ignore error 404 messages, while creating fedora json files

### PR DESCRIPTION
**Pull request by Princee215**
While creating Fedora files through ./bin/package_build.py. We face some errors while creating these files as can be seen in Issue #148. As mentioned directories are not present for following Fedora version- 

Screenshot from 2023-07-29 22-56-22

- Fedora 35, 36, 37 - Directory 8
- Fedora 38 - Directory 5

So, what I did, I added continue instead of raising an error. Therefore, if some directory is not present for some distro version we'll just ignore that and iterate over to next directory. And except that I've also added 1 in dirs string as it'll cover a wider range and if some directory is not there it'll just ignore that and you won't see any errors in the terminal.

@pleia2 and @arshPratap please take a look at it.

### CODE REVIEW

File: bin/package_build.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ def fedora():
 				req = requests.get(link)
 				data = req.text
 				if req.status_code == 404:
-					raise Exception(f"404 Directory {dirs[each]} not found")
```
Comment (arshPratap) [inline bin/package_build.py:137]: @Princee215 lets keep a verbose message to let users atleast know the issue they are facing

**Review by arshPratap — CHANGES_REQUESTED** [review]

**Comment by arshPratap** [comment]
@Princee215 looks good to me
@pleia2 thoughts ?

File: bin/package_build.py (hunk 1 above)
Comment (arshPratap) [reply bin/package_build.py:137]: @Princee215 resolved

File: bin/package_build.py [hunk 2]
```diff
@@ -134,7 +134,7 @@ def fedora():
...
+					continue
 			except Exception as e:
```
Comment (arshPratap) [inline bin/package_build.py:138]: @Princee215 let's remove this try - except block and have the try block above at [here](https://github.com/openmainframeproject/software-discovery-tool/pull/162/files#diff-6f39b12c55db3a00f6283da33cbe8fdbf0aa23ef14b767cc7ce7dec7a134654eL127)

**Comment by arshPratap** [comment]
@Princee215 lgtm.. Thanks for the contributions!

**Comment by Princee215** [comment]
@pleia2, @arshPratap I've made some changes again, as discussed with Arsh, I've put the code in try catch block, and checked for a condition were we're checking if both of the links are not working. Still I've some doubts that we should output message if some directory is not present or we should ignore it? And should we put it in a try catch block too or is it fine as we're using now!

Take a look and let me know. I'll be happy to make changes.

**Review by pleia2 — APPROVED** [review]

**Pull request merged**